### DISCUSSION TITLE
Unable to run without using "Run as administrator"

### DISCUSSION THREAD

**Discussion by shannonjryan** [Eugeny/tabby#9838] [Q&A]
Version: 1.0.211
Platform: Windows x64 10.0.22631
Plugins: none
Frontend: xterm-webgl

Installed Tabby using "Run as administrator" function in Windows 11. Once installed, attempts to run Tabby start background processes but the GUI isn't visible. Only runs correctly when the executable is started using "Run as administrator". 

I've already done the following:
- Uninstalled and reinstalled the application
- Confirmed that my user profile has full permissions
- Confirmed that "Run this program as an administrator" is not checked in Properties->Compatibility

Any suggestions would be much appreciated!

**Comment by Kisaragi-ng** [comment]
Use 'Take Ownership' regedit hack, then you take over the ownership of your Tabby installation using that context menu. once this done, tabby should function normally without run as administrator
